### ISSUE TITLE
Suggest: Autocomplete field

### ISSUE THREAD

**Issue by anhskohbo** [Codestar/codestar-framework#364]
I think everybody wants this, I intend to do this for codestars fw, but right now I'm so busy.
Create a topic and ping to @Codestar, or may be I can create this field when I have free times.

**Comment by NoahjChampion** [comment]
@anhskohbo doesn't chosen Jquery do this?

**Comment by anhskohbo** [comment]
But if I have more than 1000 posts chosen is not good way.

**Comment by NoahjChampion** [comment]
@anhskohbo ohhh, ok. You're right. 

This is why I moved the font selector to its own page because Loading chosen on the font selector slows down the page loading because of all the Google Fonts it has to identify for chosen.

**Comment by Codestar** [comment]
I suggest to use https://jqueryui.com/autocomplete/ for autocomplete field. because this jquery-ui includes with wordpress core :+1: 

"jquery-ui autocomplete" with "remote datasource" is correct way for this field idea. :smile:

**Comment by NoahjChampion** [comment]
@Codestar 
@anhskohbo 

Hi. 

I already started looking into this a lot.

I've went and found about 7-10 repos and solutions and narrowed it down to what I think is the best. - Jquery Autocomplete. I found one having I think all the Chosen multiselect features using Jquery UI. 

See demo below of little experiment... (this is without autocomplete) (ie: [editbaleSelect](https://github.com/indrimuska/jquery-editable-select)) But I did find a solution that works more like Chosen with ajax load too. I will share that later. I have to go back through my bookmarks. 

<img src="https://monosnap.com/file/gixS5VdadBHEaEliSR0JjJoCAOXJd6.png">
